src/ClientData/CaptureDataTest.cpp

```diff
@@ -271,6 +279,63 @@ TEST_F(CaptureDataTest, UpdateTimerDurationsIsCorrect) {
   EXPECT_THAT(capture_data_.GetSortedTimerDurationsForScopeId(kNotIssuedId), testing::IsNull());
 }
 
+struct FormattingForEachThreadStateSliceIntersectingTimeRangeDiscretizedTest {
```

Comment:
What does "formatting" mean here? I feel like it's the wrong word.

What about `ForEachThreadStateSliceIntersectingTimeRangeDiscretizedTestCase`?

Reply:
I took it from this example:
https://engdoc.corp.google.com/eng/doc/devguide/testing/parameterized-testing.md?cl=head&polyglot=cpp

But I can change it, I'm also not sure that I like this "formatting".

Anyway, I've already used in the CallstackData new tests so I may need to change it there as well, so it will be part of a following PR.

Reply:
It's called "Formatting" there because the tests are testing a class called `Formatter`...